user-guide/Troubleshooting/Known_issues/KI_Upgrade_fails_VerifyElasticStorageType_prerequisite.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: KI_Upgrade_fails_VerifyElasticStorageType_prerequisite
+---
+
+# Upgrade fails because of VerifyElasticStorageType.dll prerequisite
+
+## Affected versions
+
+DataMiner 10.4.0 or newer.
+
+## Cause
+
+Starting from DataMiner version 10.4.0, XML storage for profiles and resources is no longer supported. The _VerifyElasticStorageType_ prerequisite ensures that the system has successfully migrated its data to an indexing database. This verification process includes the following checks to confirm that XML storage is no longer in use:
+
+1. The prerequisite automatically passes if both the Profile Manager and Resource Manager configurations indicate that the indexing database is configured. Following the documented migration procedures should ensure that these configuration files have the correct values. See also the respective docs pages [Migrating SRM profiles to the indexing database](xref:Profile_migration_to_elastic) and [Migrating SRM resources to the indexing database](xref:Resources_migration_to_elastic).
+
+1. If the configuration file is either missing or indicates the use of XML storage, the prerequisite still passes if the storage file is absent or contains no objects. The storage files for the Resource Manager and Profile Manager are located at _Skyline Dataminer\ResourceManager\Resources.xml_ and _Skyline Dataminer\ProfileManager\Profiles.xml_ respectively. After the upgrade, both managers will automatically start using the indexing database for storage. If the XML storage file cannot be verified (due to being in use by another process or being corrupt), the prerequisite will fail.
```

Comment:
can we use 'XML storage file'  everywhere in the sentence to avoid confusion ?

Reply:
Done